### DISCUSSION TITLE
Release 0.8.0 - Second birthday of Box, Box!

### DISCUSSION THREAD

**Discussion by BrightDV** [BrightDV/BoxBox#169] [Announcements]
# Changelog

## Bugfixes 🐛

- Fix the Free Practice webview.
- Fix Circuit facts and history scraper.
- Fix the Race hub's documents parser.
- Fix the Race hub's information button.
- Maybe fix the Circuit screen, crashing just after the end of a race.
- Fix the Mixed News theming.

## Improvements 📈

- Add support for Formula E! (#152)
- Add support for downloading articles and videos from Formula 1! (#155)
- Add support for custom feeds in Mixed News. (#160)
- Add support for Pre-Season testings (only schedule, no results available).
- Improvements to the Race Hub screen.
- Official API added to the default settings.
- Update translations from Weblate.

## Dev 💻

- Add Monero donation (XMR). (#156)
- Update README (new screenshots, updated strings).
- Use new web implementation following Flutter bump.
- Add per-abi APKs to the release.
- Remove notification from video player.

## Thanks for using Box, Box!

<hr /><em>This discussion was created from the release <a href='https://github.com/BrightDV/BoxBox/releases/tag/0.8.0'>Release 0.8.0 - Second birthday of Box, Box!</a>.</em>

**Comment by serrq** [comment]
My apologizes for the "numerous features requests" 😅

**Reply by BrightDV** [reply]
No worries, it makes the app better!
